tests/parm/fd_ufs.yaml

```diff
@@ -1168,12 +1162,67 @@
        canonical_units: 1
        description: AOD
      #
+     - standard_name: Sl_sfrac
+       alias: mean_snow_area_fraction_lnd
```

Comment:
We want to be sure that we're exchanging fields as "inst" fields, so these aliases should be changed to 'inst' and the corresponding field names in ``module_cplfields`` also changed.

Reply:
@DeniseWorthen there are also other fields with mean prefix. Do you want me to update those too?

Reply:
Yes, we want to ensure that we're exchanging ``inst`` fields (see https://github.com/NOAA-EMC/fv3atm/issues/738)

Reply:
@DeniseWorthen done